mnefun/data/canonical.yml

```diff
@@ -0,0 +1,179 @@
+general:
+  work_dir: '.'
+  subjects_dir:
+  subject_indices:
+  disp_files:
+
+fetch_raw:
+  subjects:
+  structurals:
+  dates:
+  acq_ssh:
+  acq_dir:
+  acq_port:
+  run_names:
+  runs_empty:
+  subject_run_indices:
+
+scoring:
+  score:
```

Comment:
How do I evoke `score.py` from here?

Reply:
This is one of the few things that we haven't yamlized, because there isn't a great way to do it. I'd just keep it in Python for now like:

https://github.com/LABSN/mnefun/blob/master/examples/funloc/analysis_fun.py#L31-L34